napari/utils/events/_tests/test_evented_model.py

```diff
@@ -573,3 +574,93 @@ def b(self):  # pragma: no cover
 
             class Config:
                 dependencies = {'b': ['x']}
+
+
+def test_property_get_eq_operator():
+    """Test if the __eq_operators__ for properties are properly recognized"""
+
+    class Tt(EventedModel):
+        a: int = 1
+
+        @property
+        def b(self) -> float:  # pragma: no cover
+            return self.a * 2
+
+        @property
+        def c(self):  # pragma: no cover
+            return self.a * 3
+
+    assert Tt.__eq_operators__ == {'a': operator.eq, 'b': operator.eq}
+
+
+def test_reduce_event(monkeypatch):
```

Comment:
It's a bit hard to tell what this does; maybe call it `test_events_are_fired_only_if_necessary` or something like that?